src/ImageSharp/Formats/Webp/Lossy/LossyUtils.cs

```diff
@@ -199,6 +215,91 @@ private static int Vp8_Sse16xN_Avx2(Span<byte> a, Span<byte> b, int numPairs)
         return Numerics.ReduceSum(sum);
     }
 
+    [MethodImpl(InliningOptions.ShortMethod)]
+    private static int Vp8_Sse16x16_Neon(Span<byte> a, Span<byte> b)
+    {
+        Vector128<uint> sum = Vector128<uint>.Zero;
+        ref byte aRef = ref MemoryMarshal.GetReference(a);
+        ref byte bRef = ref MemoryMarshal.GetReference(b);
+        for (int y = 0; y < 16; y++)
+        {
+            sum = AccumulateSSE16Neon(
+                ref Unsafe.Add(ref aRef, y * WebpConstants.Bps),
```

Comment:
Using  pointers could allow emitting better code for address calculation. We can then increment the pointers as 
```
aPtr += WebpConstants.Bps;
bPtr += WebpConstants.Bps;
```
This would improve address calculation.
before:
```
lsl     w0, w20, #5
sxtw    x0, w0
add     x19, x19, x0
```

after:
`add     x19, x19, #32`

Reply:
The generated code looks indeed a bit better with pointers. I was not aware of that.

Here is a SharpLab gist: [Sse16x16_NeonPoniters](https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzmAPgAQEwEYCwAUJgAwAEmuAdAEoCuMCUYAplQMID2YADlADYsATgGVhANygBjFigDcRUhWr1GzNgElGwzjzFDJM+YvKVaDJqypaEQ2CmnHiplRfXXGdmA6koqAQSEwBUJFAGYKbDJ2MgBvIjJEsh47cQgEFjIpTm8EMlg8gCEeFDIAXjIw7BCksgSklKg0jOUANnzGMgA1HgAOAH0RFBZcVoAPUf6AORYcgAoRHggYAB5gAE8MgD4yCDgyReW1zZYd4ABKesT4wlrarpYpBE4hXGxelboCnZQ6MHLuo9nq93p9vlQAFo6Gp3RJCFgAMzIGxaEBoiIB8KRAFkWGAXutsRAhCgABYQfhUADiLAQ6IRwhYMBkcwg5xhsKxyJOyPpmIxuPxQkJxLJFOptPpjOZLDmFw5dwRLzIcwKZHWAJIcnVZBWZFG2vWAGojZdbrC4lcLYlfv8Kv4pFI/nR+OkWCIRABRUYzeZW621LkAVW8EAZAQAJhG5ly0Yj9hqAFRkYooc5wf0BuEYkMoMNsfxRmMY4D0hNkZOp9OZrO29k1sgAXyIDcwAHYVQVzg8ni83r0qCI/nM6wrm+bElbGs1MpR2gw8wyOnkegMhiNxpNfTAAAqcArCBZLVYo067faHE8nM5m2o3a094H9sGMH5/AGPvugr6vyHQhsIlAYwsBGKqnsmcZIhUABkgoEkSJLkpSNJ0oi0osmyt4WveWZkIBwGgXKJzJqWGIwXBwoIWKyGSmh8IynK5xYVmOG4UkSpCJ2nQahUWo6nqBrqiazFsZaE6ibUtoAg6TpgC6boet6rTbqyfJGiqSYpiU6a8hi6lzJpVb7KODbWuObHmRa46th2qqMN2QJfgOQ5gCOfz1uJlmTuJ05um0ZALvmy7dH0gzDKMEytNMsy7jkUAZCSR5HKeOx7Acx7HNsyIiaxdyfiCHw/ggb52oCvYFS+xV/kInAhAAxCkEAAOaQGQADuxIwLATVkBGUB5sAgjRCIvQAKwkCQDbgbsfIVERGSJuc0EQAqtTTaRUFgcRS3APVjUtRA7Wdd1ZDwigwKZOwI3jZN4m1BxXF5DxZB8RqAmtIawkNrl1pSfajrOq6GSKT6MWqXpGkVlpab7BtZD6YZ2nGe5q1JNZd1JO2j0OeV/aDsOJmeVaU6pH5c5lU+37fGQMmAwpXqg/MXKnjN8anSWPIbfs+XPkVJU5Q2POgiluzkBUub5gEKCZSw3OORVKVbMWSKQR5D7y8+IvAGLZAS+G/jS6ecu48L15K8z9IeYLGum1lEDACgAAiUAIpthbiCIzARgEDucPwdAZM7rvobKEAkLDJBqxa+WtAALDLqUO0HCIADKcG1wgAvbTsuwiEoIGnGdCHMUewjH8ci9nydBjwPCZxUVe5/nNd18Xpd5TbhVki8xXJDVEa4NJEYe17VDYi6TA8Pw6wAOpQBGTLdYXh6N67y9CPsq+p+nwjt/cnefN3Qi9yknARlE9rD57YDe+P/CT9Pc8L11MBNevrJJ7nLfCJvn+u9/QgrYY0SAAehAckaAQgAC0bV+qZAgFGUoyxQKwOflQa2JtCrU1tIPS+I8b6RgjDuSBsDhhP0Xq/OYp8B57ySELLBr4yC2gvjTK+o9CxEJIXA8hL8mpUP7tgWhiQsbu2vt7DhbkwD7FEewosODkZgEEe3LydQfKkxaOTQKS56GVR2LTOSQN3QM2UmDaakFYbEV0giY2lMGG9zrFaH6dCD6Vx1jIghacEH0PBgiIRFMnIJ2RG4thHjOBeM7nKS2IQMG2MCVvIe+DxG+39oHXOIdWThyCX48ucS/7byLlnPJ+d37ZPlnHXJOd/613rrsIpKEAEl1RokHRdAj4n37rg1hiSx4TygFPWe88KFvx3sXLe69f6VPybvJp/iKqtNJD3HY1CWHuNvr0/pPCl4jI/pMgBEzq7VMAdE4BZAwEQKgNA0h8DEG7BgCgwZMB0EnJadgv4nTVmEOIRcq5mzKHUNwKUzBuimF/BWSE8RUYvlCB+Q87q/Cz6CJmSI8FhDJHSJRRI+RILFFMTHEQRsQA===)